src/editors/data/redux/thunkActions/app.js

```diff
@@ -43,9 +44,15 @@ export const saveBlock = ({ content, returnToUnit }) => (dispatch) => {
   }));
 };
 
+export const fetchImages = ({ onSuccess }) => () => {
```

Comment:
We'll forgoe docstrings here for now, but when this hook gets plugged into the network we'll want to add them here-- just a note for me.

Reply:
100%